Tensile/KernelWriterAssembly.py

```diff
@@ -3065,7 +3109,21 @@ def globalReadDo(self, kernel, guardK, tP):
                         "0 idxen offset:0",\
                         "load single f16")
                   elif kernel["ProblemType"]["DataType"].isSingle():
-                    kStr += inst("buffer_load_dword", \
+                    if kernel["DirectToLds%s"%tP["tensorChar"]]:
+                      tmpVgpr = self.vgprPool.checkOut(1)
+
+                      # TODO-LDSDestBug.  Remove dummy tmpVgpr when LDS dest asm bug fixed:
```

Comment:
Looks like this refers to the above code where I commented on lds being set to 0. It would be great if there were some way to tie the need for this workaround to the version of the assembler. Of course, we would need to know the working version number to set the appropriate cutoff! Or, can we try to assemble assuming the bug is fixed and use a failure to trigger the workaround?